mu-plugins/central-hub/tests/phpunit/integration/meta-data/isOkayToSaveMetaBox.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Tests for the function is_okay_to_save_meta_box().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\Metadata;
+
+use function spiralWebDB\Metadata\is_okay_to_save_meta_box;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_IsOkayToSaveMetaBox
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @group   meta-data
+ */
+class Tests_IsOkayToSaveMetaBox extends Test_Case {
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
+		$_POST = [];
```

Comment:
@rgadon107 You  need to include `parent::tearDown()` as the first line in this method.

Both the setUp and tearDown methods extend off of the parent class.To make sure the parent's code runs, you invoke it by `parent::setUp()` or `parent::tearDown()`.

Reply:
@hellofromtonya I wasn't sure that I needed to explicitly declare that. I assumed that since the test class `Tests_IsOkayToSaveMetaBox`  _extends_ the parent class `Test_Case`, that `parent::tearDown()` would automatically run after each test method. 

I'll put it back in.